src/App.js

```diff
@@ -1,25 +1,159 @@
-import logo from './logo.svg';
-import './App.css';
+import React, { useState } from "react";
+import "./SlotMachine.css";
+
+const symbols = ["🍍", "🐠", "🥥", "🌴", "🐒", "🏴‍☠️"];
+const credits = [1, 5, 10, 20, 50, 100];
+
+function getRandomSymbol() {
+  return symbols[Math.floor(Math.random() * symbols.length)];
+}
+
+const SlotMachine = () => {
```

Comment:
I would move this out of the App.js and into src/components/SlotMachine. You want to modularize everything and put the relevant logic into that module. That way, when one module rerenders, this whole App and everything in it doesn't need to rerender. Only the relevant modules should rerender. Also, if one module breaks, it doesn't break the whole app and it is easier to debug.